shared-scripts/core/usr/bin/base-usage

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh -e
+
+cat <<EOF
+This image serves as the base image for all OpenShift v3 S2I builder images.
```

Comment:
```suggestion
This image serves as the base image for all OpenShift v3 and OpenShift v4 S2I builder images.
```

Reply:
How about just removing the mentions of versions. 
```suggestion
This image serves as the base image for all OpenShift S2I builder images.
```